scientific_library/tvb/simulator/plot/config.py

```diff
@@ -34,47 +34,63 @@
 """
 
 import os
-import numpy as np
 from datetime import datetime
 
+import numpy
+
+
+class GenericConfig(object):
+    _module_path = os.path.dirname(__file__)
+
+    # Identify and choose the Simulator, or data folder type to read.
+    MODE_H5 = "H5"
+    MODE_TVB = "TVB"
+
+
+class InputConfig(object):
+    _base_input = os.getcwd()
+
+    @property
+    def HEAD(self):
+        if self._head_folder is not None:
+            return self._head_folder
+
+        # or else, try to find tvb_data module
+        try:
+            import tvb_data
+            return os.path.dirname(tvb_data.__file__)
+        except ImportError:
+            return self._base_input
+
+    @property
+    def IS_TVB_MODE(self):
+        """Identify and choose the Input data type to use"""
+        return self._data_mode == GenericConfig.MODE_TVB
+
+    @property
+    def RAW_DATA_FOLDER(self):
+        if self._raw_data is not None:
+            return self._raw_data
+
+        return os.path.join(self._base_input, "data", "raw")
+
+    def __init__(self, head_folder=None, raw_folder=None, data_mode=GenericConfig.MODE_TVB):
+        self._head_folder = head_folder
+        self._raw_data = raw_folder
+        self._data_mode = data_mode
 
-class FiguresConfig(object):
```

Comment:
We should have here under tvb.simulator.PLOT just a configuration for Figures. This is the single purpose for this module